cardano-cli/test/cardano-cli-golden/files/golden/help/allegra_governance_drep_delegation-certificate.cli

```diff
@@ -0,0 +1,31 @@
+Usage: cardano-cli allegra governance drep delegation-certificate 
```

Comment:
I was told this is wrong, so I'm not sure if `governance drep` should appear everywhere: https://input-output-rnd.slack.com/archives/G011N23CEAE/p1691759918638949?thread_ts=1691691053.122579&cid=G011N23CEAE

Reply:
I'm following the "Cardano-CLI Feature Requirements to support Voltaire" document.

Reply:
Definitely Dreps only exist in Conway. Not in previous eras.